r/tests/testthat/test-parquet.R

```diff
@@ -234,3 +234,11 @@ test_that("ParquetFileReader $ReadRowGroup(s) methods", {
   expect_true(reader$ReadRowGroups(c(0, 1), 0) == Table$create(x = 1:20))
   expect_error(reader$ReadRowGroups(c(0, 1), 1))
 })
+
+test_that("Error messages are shown when the compression algorithm lz4/snappy
+          is not found", {
+  if (codec_is_available("snappy")) {
+    d <- read_parquet(pq_file)
+    expect_is(d, "data.frame")
+  }
+})
```

Comment:
Does this test do anything more than the one on lines 24-29 do?

Reply:
I removed this since it's pure duplication